CODEOWNERS

```diff
@@ -44,12 +43,11 @@ src/drivers/imx/**			@dbaluta
 src/drivers/dw/*			@lyakh
 
 # other libs
-src/host/*				@ranj063
 src/math/*				@singalsu
 src/ipc/*				@xiulipan @bardliao
-src/ipc/sue-ipc.c			@lyakh
+src/drivers/intel/cavs/sue-ipc.c	@lyakh
 src/lib/*				@libinyang
-src/gdb/*				@mrajwa
+src/debug/*				@mrajwa
```

Comment:
@mrajwa should be responsible just for `src/debug/gdb` stuff, debug may have later more generic stuff, that shouldn't be his concern.

Reply:
Done, thanks I wasn't sure.